### PR TITLE
fix(northlight): toolbox style updates

### PR DESCRIPTION
**Pull request by ilija-**
Two updates

1. `borderWidth` & `borderColor` are now updated with semantic tokens
2. Fixed visual bug
**From this:**
Screenshot 2024-12-23 at 16 13 13

**To this:**
Screenshot 2024-12-23 at 16 13 03

closes: DEV-17582

### CODE REVIEW

File: framework/lib/theme/components/toolbox/index.ts [hunk 1]
```diff
@@ -2,21 +2,25 @@ import { ComponentMultiStyleConfig } from '@chakra-ui/react'
...
+  } }) => ({
     container: {
       position: 'relative',
       bg: 'text.inverted',
       h: '100vh',
-      borderLeftWidth: '1px',
+      borderLeftWidth: borderWidth.xs,
       borderLeftStyle: 'solid',
-      borderLeftColor: 'gray.100',
+      borderLeftColor: color.border.default,
```
Comment (SebCodesTheWeb) [inline framework/lib/theme/components/toolbox/index.ts:16]: border.default is equivalent to gray.200, not gray.100, is this change intentional?

Comment (ilija-) [reply framework/lib/theme/components/toolbox/index.ts:16]: Yup. Part of making the movement "Figma 1:1 with NL".

Comment (SebCodesTheWeb) [reply framework/lib/theme/components/toolbox/index.ts:16]: Okay I can see in the sketches that it was supposed to be border.default from the start, nvm

**Review by SebCodesTheWeb — APPROVED** [review]

**Comment by SebCodesTheWeb** [comment]
gj
